docs/multi_process.md

```diff
@@ -312,6 +312,83 @@ and running computations. The remaining Python code examples in this tutorial
 are meant to be run on all processes simultaneously, after running
 {func}`jax.distributed.initialize`.
 
+### Kubernetes
+
+Running multi‑controller JAX on a Kubernetes cluster is almost identical in spirit to the GPU and TPU examples above: every pod runs the same Python program, JAX discovers its peers, and the cluster behaves like one giant machine.
+
+1. **Container image** – start from a JAX‑enabled image, e.g. one of the public JAX AI images on Google Artifact Registry ([TPU](https://console.cloud.google.com/artifacts/docker/cloud-tpu-images/us/jax-ai-image/tpu) / [GPU](https://console.cloud.google.com/artifacts/docker/deeplearning-images/us-central1/jax-ai-image/gpu)) or NVIDIA [NGC](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/jax) / [JAX-Toolbox](https://github.com/NVIDIA/JAX-Toolbox).
```

Comment:
Would you mind using `[text][link]` style links instead and moving these URLs to the bottom of the file?

Reply:
done